### DISCUSSION TITLE
Oauth sign in - sending custom data

### DISCUSSION THREAD

**Discussion by MamoshiSE** [nextauthjs/next-auth#5619] [Help]
### Question 💬

Is it possible to send in custom data on oAuth sign in from client side? for example on signIn('twitch'). I tried doing like signIn('twitch',  {type: 'name_update_test'})

But haven't managed to access the type parameter in nextauth.js, not sure if possible even. Sending custom data seems to work with credentials login though.

Is this possible, if so how can I access the data that is sent in from client side in the backend [...nextauth].js

### How to reproduce ☕️

Use nextauth starter template and try sending in custom data on oauth sign in method from client side

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by sushanyadav** [comment]
Hello.
any updates on this?
